README.md

```diff
@@ -1,5 +1,33 @@
 # ACCESS-ESM with **payu**
 
+## Quickstart Guide
+
+Get payu:
+
+    $ module use /g/data3/hh5/public/modules
+    $ module load git conda payu
+
+Create the laboratory:
+
+    $ mkdir -p /short/$PROJECT/$USER/access-esm
+
+Create the configuration directory:
+
+    $ mkdir -p ~/access-esm
+    $ cd ~/access-esm
+    $ git clone https://github.com/coecms/esm-pre-industrial pre-industrial
```

Comment:
What is wrong with just:
`git clone https://github.com/coecms/esm-pre-industrial`
Why rename? That could be confusing for the user later if is not the same as this repo